src/Features/Core/Portable/xlf/FeaturesResources.cs.xlf

```diff
@@ -4200,6 +4200,11 @@ Chcete pokračovat?</target>
         <target state="translated">přístupový objekt události</target>
         <note />
       </trans-unit>
+      <trans-unit id="required_property">
+        <source>required property</source>
+        <target state="new">požadovaná vlastnost</target>
```

Comment:
I've used the machine translation and can't be sure about the accuracy.

Reply:
You don't need to translate anything. The localization team will take care of that. You just add the item in English and that's it

Reply:
reverted the localizations to English